scripts/install_all.sh

```diff
@@ -182,16 +183,6 @@ if ! command -v ip > /dev/null; then
     fi
 fi
 
-if ! test -e ../.git/hooks/pre-commit > /dev/null; then
-    if confirm "Would you like to configure a pre-commit hook for this project to run black (code formatter) prior to each commit?"; then
-        echo "Installing pre-commit hook to auto-format code."
-        cp hooks/autoformat.sh ../.git/hooks/pre-commit
-        chmod +x ../.git/hooks/pre-commit
-    else
-        echo "Not installing pre-commit hook to auto-format code."
-    fi
-fi
```

Comment:
Why's this being removed?

Reply:
This doesn't actually ever run, because by the time we get to this line `pre-commit` has already installed its own git hook.

Reply:
Ahh gotcha!

Reply:
I think this was how we installed git hooks before pre-commit?